.travis.yml

```diff
@@ -25,12 +25,13 @@ before_install:
 install:
   - cd core 
   - cd new-gui
-  - npm install
+  - npm install --unsafe-perm
   - cd ..
 
 script:
-  - mvn test --batch-mode
+  - mvn clean install
```

Comment:
why change `mvn test ` to `mvn clean install`, and don't remove `--batch-mode`, this option makes CI output looks better.